### DISCUSSION TITLE
Do data ever really get deleted?

### DISCUSSION THREAD

**Discussion by sjgknight** [sjgknight/DSI_collection_sheets#22] [Q&A] [answered]
I am sure all deleted data are still stored in separate server, eventhough companies argued that they do not retain data (like snapchat, telegram)

**Comment by passiona** [comment, marked as answer]
Great question, that will vary between companies, based on their practices and if they have reporting requirements or record-holding regulations. For example, Google has entire team(s) dedicated to deleting user account records. However, there are some records that Google is required to retain for a specified period, namely financial transactions.

As a result of differing practices, some companies will use the fact they actually delete records as a point of difference in their offering. It is possible to completely delete data and alternative approaches include marking records as inactive.
